moped-editor/src/queries/project.js

```diff
@@ -25,7 +25,6 @@ export const ADD_PROJECT = gql`
       }
       moped_project_types {
         project_type_id
-        status_id
```

Comment:
I missed removing this when updating `moped_project_types` with `is_deleted`

Reply:
👍 , but just checking that I'm not missing something -- this is a tag-along change that could have happened in another PR addressing another issue? No worries if so, just checking my understanding.

Reply:
That is totally right. ✅  I should have linked the issue and PR - this change should have happened for https://github.com/cityofaustin/atd-data-tech/issues/9250 and merged in with https://github.com/cityofaustin/atd-moped/pull/660